### DISCUSSION TITLE
no/

### DISCUSSION THREAD

**Discussion by giscus** [SX-Code/blog-comments#6] [Announcements]
# about/

swcode这是我的简介      关于作者关于本站友情链接  个人信息 毕业院校：河南科技大学，软件工程(本科)，2018年毕业 博客园： GitHub： Email： QQ：  工作经历中国地质大学（武汉）（2022 - 至今）攻读方向  星载单光子数据处理 软件开发方向  本站建站初衷是存放学习笔记、个人博客以及一些小技巧。

https://sx-code.github.io/about/

**Comment by SX-Code** [comment]
Stellar主题更新，更换博客评论系统为 [giscus](https://giscus.app/zh-CN)
